README.md

```diff
@@ -14,6 +14,7 @@ Single-header CLI library intended for use in embedded systems (like STM32 or Ar
 * Live autocompletion (see demo above, can be disabled)
 * Tab (jump to end of current autocompletion) and backspace (remove char) support
 * History support (navigate with up and down keypress)
+* Limited cursor support
```

Comment:
Why is it limited? I think it will be better to explain, for example (left and right keypress to navigate inside input)

Reply:
Added suggested comment